tools/StaticAnalysis/Exceptions/Az.ManagedNetwork/SignatureIssues.csv

```diff
@@ -1,2 +0,0 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
seems liek this file should be completedlt removed.  Also, the 'PS-VSPrompt.lnk file above should not be included in this PR.

Reply:
@markcowl
Yes, `SignatureIssues.csv` is indeed deleted.

"PS-VSPrompt.lnk" was deleted by accident in the original PR. This PR will recover it back.